baseapp/block_gas_test.go

```diff
@@ -173,7 +173,7 @@ func TestBaseApp_BlockGas(t *testing.T) {
 				require.Equal(t, []byte("ok"), okValue)
 			}
 			// check block gas is always consumed
-			baseGas := uint64(51732) // baseGas is the gas consumed before tx msg
+			baseGas := uint64(50702) // baseGas is the gas consumed before tx msg
```

Comment:
This test relies on some bank balances modifications, collections happens to be a little more gas efficient with respect to this.